README.md

```diff
@@ -93,6 +93,11 @@ location.
 # Installation
 
 ## Installing with pip
+Check out the Mac setup guide for vrs-python [here](https://github.com/ga4gh/vrs-python/blob/main/docs/setup_help/m1_mac_setup.md). This is a crucial step before diving into UTA. Once you've completed the setup, proceed with configuring the PostgreSQL database for the Unified Transcriptome Annotation (UTA) application  [here](https://github.com/ga4gh/vrs-python/blob/main/docs/setup_help/uta_installation.md).   
+
+Before installing the `ga4gh.vrs[extras]` package, ensure that **PostgreSQL** is installed on your system. You can download and install PostgreSQL from [here](https://www.postgresql.org/download/).
```

Comment:
Should we also include installation via homebrew?

Not sure which version we want to maintain:

```
brew formulae | grep postgresql@
```

Reply:
I guess the m1_mac_setup.md has the brew instructions 
I don't see this instruction there:
```
brew formulae | grep postgresql@
```
It's still not clear to me how to address the installation instructions of UTA based on the provided info.

Reply:
Right the command was just listing possible version. I am not sure which ones out of these we should maintain. But I think homebrew could be given as another option in the repo readme. 

We can ignore my other comment about the other docs and address in a separate issue.

Reply:
So how should we proceed with homebrew and should we keep the dedicated uta instruction or just link it to the official uta guide ?

Reply:
Let's stick with your original commit (sorry) and we can make a new issues (if they do not already exist):
* Add information on which versions are supported / option to install via homebrew
* Reference setup help docs in readme

Reply:
No problem I'll make the changes soon.

Thanks and Happy New Year 🎉💯

Reply:
If any additional information is required or any omissions are necessary, please feel free to let me know .